csaf_2.1/json_schema/csaf_json_schema.json

```diff
@@ -855,7 +855,7 @@
             },
             "initial_release_date": {
               "title": "Initial release date",
-              "description": "The date when this document was first published.",
+              "description": "The date when this document was first released to the intended target group.",
```

Comment:
Intended or specified? I prefer the latter ...

Reply:
@sthagen Do we also need to change the `sharing_group`? I would say no, just flagging for attention.

Reply:
[39a992a](https://github.com/oasis-tcs/csaf/pull/873/commits/39a992af59e2a47933a360e9724a3ff682af5e9c)